### DISCUSSION TITLE
Linting

### DISCUSSION THREAD

**Discussion by AndreyDmitriev39r** [Bill-1991/Scheduler#22] [General]
@Bill-1991 what exactly is wrong with linting(you mentioned some issues with that on the server)?

Point to the flow you used for trying it out, and please describe exact probs with that.

**Comment by Bill-1991** [comment]
https://discordjs.guide/preparations/setting-up-a-linter.html#installing-a-code-editor
There is nothing wrong with it, but just copying over the rules from discord will have many rules that take time to fix.
If we gonna set it up, we 'll have to setup our rules first and not a ton of them i think

**Comment by AndreyDmitriev39r** [comment]
> https://discordjs.guide/preparations/setting-up-a-linter.html#installing-a-code-editor There is nothing wrong with it, but just copying over the rules from discord will have many rules that take time to fix. If we gonna set it up, we 'll have to setup our rules first and not a ton of them i think

Makes sense, but it would require some digging into linting options.
Linters are dev tool to make life easier, so I dunno if we wanna handpick linting rules.
I think linters is where you can rely on some pre-written templates without much concerns.
